src/vs/workbench/browser/web.simpleservices.ts

```diff
@@ -71,6 +71,16 @@ export class SimpleExtensionTipsService implements IExtensionTipsService {
 	getAllIgnoredRecommendations(): { global: string[]; workspace: string[]; } {
 		return { global: [], workspace: [] };
 	}
+
+	// {{SQL CARBON EDIT}}
+	getRecommendedExtensionsByScenario(scenarioType: string): Promise<IExtensionRecommendation[]> {
+		return Promise.resolve([]);
+	}
+
+	promptRecommendedExtensionsByScenario(scenarioType: string): void {
+		return;
+	}
+	// End of {{SQL CARBON EDIT}}
```

Comment:
you might copied form my PR, let’s make this consistent, use {{SQL CARBON EDIT}} - End, just like other places in your PR